packages/playground/blueprints/src/lib/compile.ts

```diff
@@ -192,21 +202,8 @@ export function compileBlueprint(
 		}
 		const wpCliInstallStep: WriteFileStep<FileReference> = {
 			step: 'writeFile',
-			data: {
-				resource: 'url',
-				/**
-				 * Use compression for downloading the wp-cli.phar file.
-				 * The official release, hosted at raw.githubusercontent.com, is ~7MB
-				 * and the transfer is uncompressed. playground.wordpress.net supports
-				 * transfer compression and only transmits ~1.4MB.
-				 *
-				 * @TODO: minify the wp-cli.phar file. It can be as small as 1MB when all the
-				 *        whitespaces and are removed, and even 500KB when libraries
-				 *        like the JavaScript parser or Composer are removed.
-				 */
-				url: 'https://playground.wordpress.net/wp-cli.phar',
-			},
-			path: '/tmp/wp-cli.phar',
+			data: wpCliResource,
+			path: wpCliPath,
```

Comment:
I attempted to reuse data here, but it doesn't feel great.

Reply:
Yeah let's not fetch that plugin in the JS API handler at all. I realize that's inconsistent, let's solve for that later on as we migrate to the PHP library.